FlowCrypt/Controllers/Inbox/InboxProviders.swift

```diff
@@ -50,11 +50,16 @@ class InboxMessageListProvider: InboxDataProvider {
 
     override func fetchInboxItems(using context: FetchMessageContext, userEmail: String) async throws -> InboxContext {
         let result = try await provider.fetchMessages(using: context)
-        let inboxData = result.messages.map { InboxRenderable(message: $0) }
+
+        let inboxData = result.messages
+            .sorted(by: { $0.date > $1.date })
+            .map(InboxRenderable.init)
```

Comment:
I was hoping to avoid doing that (because I want to be able to change order by sending a different parameter to Gmail list fetch, and not having to order in two places - always once in Gmail API and once manually..).

Do the messages from Gmail API list come correctly or wrongly ordered?

Reply:
I removed additional sorting here, as API response returns right order.